scrapers/orgs/getOrgsData.go

```diff
@@ -0,0 +1,121 @@
+package orgs
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"github.com/go-resty/resty/v2"
+	"github.com/joho/godotenv"
+	"encoding/csv"
+	"errors"
+	"eshaanagg/lfx/database"
+	"eshaanagg/lfx/database/handlers"
+	"strings"
+)
+type orgsResponse struct {
+	content      string
+    skills       []string
+    term         string
+    website      string
+}
+
+const (
+	apiEndpoint= "https://api.openai.com/v1/chat/completions"
+)
+
+func getOrgsData(orgsID string)(*orgsResponse, error){
+	empty:= orgsResponse{
+		content:" ",
+		skills:[]string{""},
+		term: "0",
+		website:"",
+	}
+	err := godotenv.Load(".env")
+	if err != nil {
+		log.Fatalf("Some error occured. Err: %s", err)
+		return &empty, err
+	}
+	apiKey:= os.Getenv("CHATGPT_API_KEY")
+	client:= resty.New()
+	response,err:=client.R().
+	SetHeader("Authorization","Bearer"+apiKey).
+	SetHeader("Content-Type","application/json").
+	SetBody(map[string]interface{}{
+		"model": "gpt-3.5-turbo",
+		"messages":[]interface{}{map[string]interface{}{"role":"system","content":"Generate a description for XYZ organization that comes in LFX Mentorship, It should tell the perks of contributing to their organization in about 100 words divided to 2 or 3 paragraphs unequally."}},
+		"max_tokens":100,
+	}).Post(apiEndpoint)
+	if err != nil {
+		log.Fatalf("Error while sending the response, Be Patient")
+		return &empty, err
+	}
+	body:=response.Body()
+	var data map[string]interface{}
+	err= json.Unmarshal(body, &data)
+	if err!= nil {
+		fmt.Println("Error while generating your response", err)
+		return &empty, err
+	}
+	choices, ok:=data["choices"].([]interface{})
+	if !ok || len(choices) ==0{
+		fmt.Println("No response choices found")
+		return &empty, err
+	}
+	content := choices[0].
+	(map[string]interface{})["message"].(map[string]interface{})["content"].(string)
```

Comment:
You should avoid `map[string]interface{}` as much as possible. But since it will be used only once, it is fine for now. But leaving this comment as a reference for future.